app/screens/sellers/SellingItemList.java

```diff
@@ -1,32 +1,26 @@
-package icebase.app;
+package icebase.app.screens.sellers;
 
 import java.util.Scanner;
 
-public class ItemList {
-    ItemList() {
-
-    }
+import icebase.app.App;
+import icebase.app.MenuTitle;
+import icebase.app.screens.Screen;
+import java.util.Scanner;
 
-    public static void buyItem(String item) {
-        System.out.println(item);
-        // display other item attributes
-    }
+public class SellingItemList implements Screen {
 
-    public static void main(String[] args) {
-        Scanner sc = new Scanner(System.in);
+    public void display() {
+        Scanner sc = App.sc;
         int choice;
         String storeName = "STORE X"; // stand-in value, should be set from constructor
         String categoryName = "CATEGORY X"; // stand-in value, should be set from constructor
-        String[] itemList = { "Item A", "Item B", "Item C" }; // fetch from icebase
+        String[] itemList = { "Item A", "Item B", "Item C", "Return" }; // fetch from icebase
         int range = itemList.length;
```

Comment:
tenks² for improving,, paki remove nalang this po or ako nalang when i pr?  hehehhuhhu,,,